packages/notification-managment-spa/README.md

```diff
@@ -0,0 +1,34 @@
+# Welcome to Notification Management SPA 👋
+![Version](https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000)
```

Comment:
Give some more information regarding the SPA

Reply:
This notification management SPA will enable other SPA-owners to create multiple notification templates if required. They will also be able to configure the notifications they receive from SPAs, a front-end for the notification microservice. @riginoommen